$NetBSD: patch-ew,v 1.1 2010/09/10 03:29:01 taca Exp $

* r26583: don't use O_NOFOLLOW because /dev/urandom is a symlink in OpenSolaris.

--- random.c.orig	2009-01-17 03:18:21.000000000 +0000
+++ random.c
@@ -280,9 +280,6 @@ random_seed()
 #ifdef O_NOCTTY
             |O_NOCTTY
 #endif
-#ifdef O_NOFOLLOW
-            |O_NOFOLLOW
-#endif
             )) >= 0) {
         if (fstat(fd, &statbuf) == 0 && S_ISCHR(statbuf.st_mode)) {
             read(fd, seed, seed_len);
